tests/test_lib_audit.py

```diff
@@ -193,3 +194,31 @@ def test_06_truncate_data(self):
                          column_length.get("serial"))
         self.assertEqual(len(self.Audit.audit_data.get("token_type")),
                          column_length.get("token_type"))
+
+    def test_07_sign_non_ascii_entry(self):
+        # Log a username as unicode with a non-ascii character
+        self.Audit.log({"serial": "1234",
+                        "action": "token/assign",
+                        "success": True,
+                        "user": u"kölbel"})
+        self.Audit.finalize_log()
+        audit_log = self.Audit.search({"user": u"kölbel"})
+        self.assertEqual(audit_log.total, 1)
+        self.assertEqual(audit_log.auditdata[0].get("user"), u"kölbel")
+        self.assertEqual(audit_log.auditdata[0].get("sig_check"), "OK")
+
+        # Log a username as utf-8 with a non-ascii character
+        username = u"kölbel".encode("utf-8")
+        # hm, even if we should log a utf8 username...
+        self.Audit.log({"serial": "1234",
+                        "action": "token/assign",
+                        "success": True,
+                        "user": username})
```

Comment:
Hmm, to be honest, I think passing an UTF8-encoded bytestring to ``log`` should be illegal. In other words, I think optimally ``log`` should only accept ``unicode`` (or, in  ASCII bytestrings which can be implicitly converted to unicode).
What do you think?